### PR TITLE
Feature/new matching api

### PR DESCRIPTION
**Pull request by mspector**
How to match ECCV reviewers:

1. Ensure that all researcher profiles that we know about have equivalent profiles in OpenReview.
2. Run upload-reviewers.py. This will:
- Detect any missing profiles (there should be none, when we run this in the real conference)
- Create the cv-foundation.org/ECCV/2018/Conference/Reviewers group
3. Run setup-matching.py. This will create the new, reviewer-specific metadata invitation.
4. Run download-profiles.py to get the profiles.pkl file
5. Run download-reviewer_profiles.py to get the cmt_conflicts.pkl file
5. Run setup-metadata-reviewers.py. This will generate the metadata required to do the match.
6. Run match-reviewers.py. Make sure to adjust the configuration directly in the file, if necessary

### CODE REVIEW

File: venues/cv-foundation.org/ECCV/2018/Conference/python/setup-matching.py [hunk 1]
```diff
@@ -35,6 +35,28 @@
     }
 }))
 
+metadata_reviewers_inv = client.post_invitation(openreview.Invitation(**{
+    'id': 'cv-foundation.org/ECCV/2018/Conference/-/Paper_Metadata/Reviewers',
```
Comment (melisabok) [inline venues/cv-foundation.org/ECCV/2018/Conference/python/setup-matching.py]: Why not call the group like this:

cv-foundation.org/ECCV/2018/Conference/Reviewers/-/Paper_Metadata?

Comment (mspector) [reply venues/cv-foundation.org/ECCV/2018/Conference/python/setup-matching.py]: good idea. that would work

**Comment by melisabok** [comment]
I found a conflict that is a gmail address?

screen shot 2018-04-05 at 7 07 51 pm

**Comment by melisabok** [comment]
Well the reviewer is the author of the paper :)

**Comment by melisabok** [comment]
I could run the matcher changing the maxpapers to 20. I see that finalScore should be the same than tpmsScore, see image attached:

screen shot 2018-04-05 at 8 24 42 pm

It seems that finalScore is the tpmsScore divided by 3.

**Comment by melisabok** [comment]
After ECCV we should discuss the scores and constraints in more detail. I’m not yet sure if they should be separated or not. Thanks!

**Comment by mspector** [comment]
I'm working on a branch where the conflicts and scores are separated. it will be separate from this branch, so this can continue working as intended

**Comment by melisabok** [comment]
Ok, but that change implies to change the matching code, right?

**Comment by mspector** [comment]
Yes, I have a corresponding branch in the matching repo too.

**Pull request merged**